### ISSUE TITLE
AWS Clouddriver NoSuchMethodError

### ISSUE THREAD

**Issue by maggieneterval** [spinnaker/spinnaker#4731]
### Issue Summary:
On master, Clouddriver with AWS accounts enabled is printing a bunch of these warnings. Not sure if we need to upgrade the library per the error message or if anyone has encountered another workaround?

```
spinnaker/spin-clouddriver-caching-658497b884-dhhhl[clouddriver-caching]: 2019-08-07 15:42:52.085  WARN 1 --- [utionAction-103] c.a.http.apache.utils.ApacheUtils        : NoSuchMethodError was thrown when disabling normalizeUri. This indicates you are using an old version (< 4.5.8) of Apache http client. It is recommended to use http client version >= 4.5.9 to avoid the breaking change introduced in apache client 4.5.7 and the latency in exspinnaker/spin-clouddriver-caching-658497b884-dhhhl[clouddriver-caching]: 2019-08-07 15:42:52.086  WARN 1 --- [cutionAction-23] c.a.http.apache.utils.ApacheUtils        : NoSuchMethodError was thrown when disabling normalizeUri. This indicates you are using an old version (< 4.5.8) of Apache http client. It is recommended to use http client version >= 4.5.9 to avoid the breaking change introduced in apache client 4.5.7 and the latency in exception handling. See https://github.com/aws/aws-sdk-java/issues/1919 for more information
spinnaker/spin-clouddriver-caching-658497b884-dhhhl[clouddriver-caching]: 2019-08-07 15:42:52.086  WARN 1 --- [utionAction-103] c.a.http.apache.utils.ApacheUtils        : NoSuchMethodError was thrown when disabling normalizeUri. This indicates you are using an old version (< 4.5.8) of Apache http client. It is recommended to use http client version >= 4.5.9 to avoid the breaking change introduced in apache client 4.5.7 and the latency in exception handling. See https://github.com/aws/aws-sdk-java/issues/1919 for more information
```

### Cloud Provider(s):
AWS

### Feature Area:
Clouddriver

**Comment by clanesf** [comment]
I'm getting a *ton* of these upgrading one of our 1.14.X systems to 1.15.2 using Clouddriver build
`gcr.io/spinnaker-marketplace/clouddriver:6.1.0-20190812153835`

**Comment by dmai-apixio** [comment]
i also get a ton of that message on version `1.15.3`
```2019-09-06 21:48:49.051  WARN 1 --- [cutionAction-13] c.a.http.apache.utils.ApacheUtils        : NoSuchMethodError was thrown when disabling normalizeUri. This indicates you are using an old version (< 4.5.8) of Apache http client. It is recommended to use http client version >= 4.5.9 to avoid the breaking change introduced in apache client 4.5.7 and the latency in exception handling. See https://github.com/aws/aws-sdk-java/issues/1919 for more information````

**Comment by iiro** [comment]
I can confirm that this is fixed in 1.16 (I've upgraded us to to the release candidate - and this is log-flooding vanished).

**Comment by clanesf** [comment]
@maggieneterval @ethanfrogers - Any chance we can cherry pick the fix for this back to 1.15.X so that the next release will fix this? As of right now 1.15.3 and the AWS provider throw this warning so often that it's very hard to run them together.

**Comment by plumpy** [comment]
@clanesf Please see my reply to your comment on spinnaker/clouddriver#3932

Unfortunately, this is not something that can be safely cherry-picked.
